cclib/combinator/combinator.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass
-from cclib.parser_properties import scfenergies
+import cclib.parser_properties as cpp
```

Comment:
"This triggers me for obvious reasons"
/click resolved conversation/
"Apparently I'm not the only one"